src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
```

Comment:
- [ ] You must have forgotten to import categoriesSlice as you did for booksReducer.